aspnetcore/fundamentals/middleware/index.md

```diff
@@ -91,15 +91,16 @@ The order that middleware components are added in the `Program.cs` file defines
 
 The following highlighted code in `Program.cs` adds security-related middleware components in the typical recommended order:
 
-[!code-csharp[](index/snapshot/Program60All3.cs?highlight=19-43)]
+[!code-csharp[](index/snapshot/Program70All3.cs?highlight=19-44)]
 
 In the preceding code:
 
 * Middleware that is not added when creating a new web app with [individual users accounts](xref:security/authentication/identity) is commented out.
 * Not every middleware appears in this exact order, but many do. For example:
   * `UseCors`, `UseAuthentication`, and `UseAuthorization` must appear in the order shown.
   * `UseCors` currently must appear before `UseResponseCaching`. This requirement is explained in [GitHub issue dotnet/aspnetcore #23218](https://github.com/dotnet/aspnetcore/issues/23218).
-  * `UseRequestLocalization` must appear before any middleware that might check the request culture (for example, `app.UseMvcWithDefaultRoute()`).
+  * `UseRequestLocalization` must appear before any middleware that might check the request culture, for example, `app.UseMvcWithDefaultRoute`.
```

Comment:
nit: `UseMvcWithDefaultRoute` is a very old API that has been replaced with endpoint routing. I think a more relevant example is needed.

Sorry, I don't know what middleware uses request culture off the top of my head.